pkg/dataloader/releaseloader/releasesync.go

```diff
@@ -31,21 +31,21 @@ type ReleaseLoader struct {
 	httpClient    *http.Client
 	releases      []string
 	architectures []string
+	platforms     []PlatformRelease
 	errors        []error
 }
 
+// Backwards compatibility for the old loader
 func New(dbc *db.DB, releases, architectures []string) *ReleaseLoader {
-	releaseStreams := make([]string, 0)
-	for _, release := range releases {
-		for _, stream := range []string{"nightly", "ci"} {
-			releaseStreams = append(releaseStreams, fmt.Sprintf("%s.0-0.%s", release, stream))
-		}
+	platformReleases, err := GetPlatformReleases("all")
+	if err != nil {
+		panic(err)
 	}
```

Comment:
if there isn't any way to configure which ones go in the loader, why even have a method call? just hardcode the result.